ramos/pool.py

```diff
@@ -35,11 +35,11 @@ def all(cls, *args, **kwargs):
 
         return [
             backend_class.create(*args, **kwargs)
-            for backend_class in cls._get_backends_classes()
+            for backend_class in cls.all_lazy()
         ]
 
     @classmethod
-    def _get_backends_classes(cls):
+    def all_lazy(cls):
```

Comment:
You may consider this a nitpick, but what do you think about naming it `all_classes` or something like that? I understand how "lazy" here refers to "it's not being instantiated", but lazy values have other meanings elsewhere too, which don't apply here.

Technically speaking, it's not returning lazy objects or data, it's returning classes

Reply:
Wow good point @lsmag you're right

Reply:
to me it's a nitpick, feel free to ignore my original comment

Reply:
No, it makes sense, I changed